tuf/repository_tool.py

```diff
@@ -2526,11 +2523,15 @@ def delegate_hashed_bins(self, list_of_targets, keys_of_hashed_bins,
       ordered_roles.append(role)
 
     for target_path in list_of_targets:
+      # Check if the target path is relative and normalize it. File's existence
```

Comment:
`_check_path()` no longer does any normalising, right? Could you update the comment here?

Reply:
Fixed during merge in https://github.com/theupdateframework/tuf/pull/1008/commits/ba8ffd218da8e44f1266a09b63c462b34f7f183d